ingestclient/plugins/zarr.py

```diff
@@ -72,18 +73,24 @@ def setup(self, parameters):
 
         Args:
             parameters (dict): Parameters for the dataset to be processed. Must
-                include keys "gs_bucket" (the name of the FQ bucket file path,
-                for example `"gs://X/Y/Z/data.zarr"`) and "volume_name", which
-                is the name of the volume in the zarr file (e.g. "raw")
+                include the following keys:
+                "backend" : [S3, GCS] for Amazon S3 and Google Cloud Service, respectively.
```

Comment:
could we do this with `bucket.startswith("s3://")` instead?

Reply:
Can you include examples in the docstring so the user knows you're looking for either "s3://..." or "gs://..."?